### PR TITLE
EchoRequestHandler

### PR DESCRIPTION
**Pull request by ryanvoong**

### CODE REVIEW

File: header.cc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include "header.h"
+
+std::string Header::to_string() {
```
Comment (jasonsliu) [inline header.cc:3]: Maybe we can just inline this into header.h?

File: request_handler.cc [hunk 2]
```diff
@@ -1,11 +1,14 @@
 #include "request_handler.h"
 
 void EchoRequestHandler::handle_request(const Request &request, 
-                                        Response *response) {
-
+                                        Response &response) {
+  response.status = 200;
```
Comment (jasonsliu) [inline request_handler.cc:5]: Is there ever a case we don't want to return 200?

**Review by jasonsliu — COMMENTED** [review]
Overall, looks good. Feel free to merge after you address comments.

File: header.cc (hunk 1 above)
Comment (ryanvoong) [reply header.cc:3]: Is it good practice to do that?

File: request_handler.cc (hunk 2 above)
Comment (ryanvoong) [reply request_handler.cc:5]: I don't think so. none of the other http response codes are possible

**Pull request merged**